### PR TITLE
Added BMS Service to SD2

### PR DESCRIPTION
**Pull request by usama2490**
Added BMS Service configurations to SD2 files and also renamed "bms_skip" var to "bms" only wherever needed.

### CODE REVIEW

**Comment by vladimirhasko** [comment]
recheck

**Comment by vladimirhasko** [comment]
recheck

File: config.yaml [hunk 1]
```diff
@@ -99,7 +99,7 @@ matrix:
           - antiddos_skip_bad_type
           - as
           - as_skip_v1
-          - bms_skip
+          - bms
```
Comment (vladimirhasko) [inline config.yaml:102]: also remove bms_skip entries from eu-nl to eu-nl and from eu-de to eu-nl monitoring scopes

**Review by vladimirhasko — CHANGES_REQUESTED** [review]
also remove EU-NL BMS entry from sdb_prod/catalog.yaml

File: config.yaml (hunk 1 above)
Comment (usama2490) [reply config.yaml:102]: The requested changes are done, please review again.

**Review by vladimirhasko — DISMISSED** [review]

**Comment by vladimirhasko** [comment]
recheck

**Review by vladimirhasko — APPROVED** [review]

**Pull request merged**